### DISCUSSION TITLE
Simple notifications around traffic spikes

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#172] [Anything Else]
"It would be great if Plausible could notify me if I get a sudden influx of traffic to my site. This is nice, actionable information to have because it usually means you’ve been linked to by a bigger site and there’s a conversation happening about your content that you can take part in (and amplify).

“Simple” means stick with an email notification sent to a list of addresses. No push notifications or anything like that. Detection could be as straight forward as a user-configured threshold (more than N concurrent visitors).

Sounds good to me. I’d find it useful for our own stats on plausible.io as well.

I like the simplicity of the approach laid our here. Just trying to think of edge cases: Let’s say I’ve configured to be notified when we have more than 15 concurrent visitors. What if the number of concurrent visitors stays above 15 for multiple days? Do I get one spike notification or one every day/hour?

I think it’s different strokes for different folks. Maybe a radio button when you set up the notification that has 3 options:

    Notify me once and only once
    Notify me hourly until traffic subsides
    Notify me daily until traffic subsides

You will probably want to put a little smarts in the notifier so that it buffers up N consecutive readings over the threshold before it triggers a notification. Otherwise if you have it set to 15 visitors and you are at 14, then 16, then 14, then 16, etc you don’t annoy people.

The challenge with notifications is always reducing the noise so that they’re actually useful. Too many and people end up disabling them altogether."

([imported from the old roadmap](https://plausible.nolt.io/140))

**Comment by jerodsanto** [comment]
**Good news**: Our Go Time [episode about HealthCare.gov](https://changelog.com/gotime/154) spent most of the day yesterday on the Hacker News homepage, on to pof /r/golang, etc., YAY!

**Bad news**: We didn't find out about it until I randomly checked Plausible during a meeting and saw 500+ current visitors

Is this feature on your radar? Because we'd LOVE to have it (and are pretty sure other media/content sites would as well) 🙏

**Reply by tcurdt** [reply]
@jerodsanto Did you guys use a self-hosted or cloud version?
I am searching for scalability information of plausible. I am wondering if a HN influx could bring a cheap VPS to its knees.

**Reply by jerodsanto** [reply]
we use the cloud service provided by Uku and his team. Not self-hosted.

**Reply by ukutaht** [reply]
Sorry, we haven't load tested on cheap/small VPS instances. Our internal testing is geared more towards our cloud offering which is running on larger servers than you would rent for private use.

My intuition says that a small VPS would be able to handle a HN spike just fine but I can't say that for sure without testing. If you want to be sure you can always use the hosted version which we keep overprovisioned to make sure we don't go down with a spike.

**Comment by ukutaht** [comment]
@jerodsanto I saw you on HN, congrats!

TBH we haven't had tons of requests for notifications but these nudges are definitely helpful in pushing it up the priority list. We'll discuss it with Marko later today.

What do you think about streaming a pairing session on this? Could be fun

**Comment by jerodsanto** [comment]
I love that idea! Let's do it. Switch to email to coordinate?

**Comment by ukutaht** [comment]
Sure, hit me up on uku@plausible.io

**Comment by metmarkosaric** [comment]
Notifications for traffic spikes are now live! Here's how to enable them on your account: https://docs.plausible.io/traffic-spikes
